manipulate_pim_data/rule/add_custom_action_rule_engine.rst

```diff
@@ -380,7 +380,7 @@ You also need to override the rule file for the datagrid with your template.
                 conditions:
                     label: pimee_catalog_rule.datagrid.rule-grid.column.conditions
                     type: twig
-                    template: PimEnterpriseCatalogRuleBundle:Rule:_conditions.html.twig
+                    template: AkeneoPimRuleEngineBundle:Rule:_conditions.html.twig
```

Comment:
```suggestion
                    template: AkeneoPimAutomationRuleEngineBundle:Rule:_conditions.html.twig
```
I feel the right namespace is this one

Reply:
Nop --> cf https://github.com/akeneo/pim-enterprise-dev/blob/3.0/src/Akeneo/Pim/Automation/RuleEngine/Bundle/Resources/config/datagrid/rule.yml#L17